sync2jira/upstream_issue.py

```diff
@@ -85,8 +85,8 @@ def handle_github_message(msg, config, pr_filter=True):
         if key == 'labels':
             # special handling for label: we look for it in the list of msg labels
             actual = [label['name'] for label in msg['msg']['issue']['labels']]
-            if expected not in actual:
-                log.debug("Label %s not set on issue: %s", expected, upstream)
+            intersection = set(actual) & set(expected)
+            if not intersection:
                 return None
```

Comment:
Rather than instantiating the `list` so that it can be used to construct a `set`, why not just instantiate `actual` _as_ a `set`?  And, rather than asking "not intersection" how about asking "if disjoint"?
```python
            actual = {label['name'] for label in msg['msg']['issue']['labels']}
            if actual.isdisjoint(expected):
                return None
```
(Note that `isdisjoint()` accepts an iterable, so there is no need to instantiate a `set` from `expected`.)

Reply:
Sure, I had just copied the old pagure implementation from the issue, but of course this is much better